hipochain/README.md

```diff
@@ -0,0 +1,53 @@
+# Hipochain
+## Objectives
+* send transactions.
+* parse transactions.
+* implement ledgering.
+
+## Assumptions
+* No consensus.
+* No peer to peer gossip.
+* No authentication.
+* There exists a single node which is a Redis client at the local network.
```

Comment:
I think instead of considering the redis client as a node we should consider all of the computers that runs the validator script as a node.
In our case, Redis was more like a tool which is used for broadcasting the transactions to the nodes.

What do you think?

Reply:
Do you think we should explicitly state what we've implemented was a `node`? If not, what do you think about the below?

```suggestion
* There exists a Redis client which is used for broadcasting the transactions.
```

Reply:
I think your suggestions is enough. 
Maybe we can explicitly mention about the Nodes.

Reply:
My suggestion was actually **not** to use Node term **anywhere**, since I'm not confident enough to state what we've implemented was a Node. That's why I asked the first question [above](https://github.com/Hipo/backend-challenges/pull/8#discussion_r878029290).

From your reply, I understand that:
- You want to keep the suggested change.
- You also want to mention what is being implemented is actually a Node. And have an explanation what Node is.

Am I right? @cemonel